packages/preview-middleware-client/src/adp/control-utils.ts

```diff
@@ -0,0 +1,484 @@
+/* eslint-disable consistent-return */
```

Comment:
Please remove this and any other eslint-disable statements and fix the liniting issues properly